source/services/encryption/PasswordStrategy.ts

```diff
@@ -0,0 +1,162 @@
+import { NativeModules } from "react-native";
+import { EncryptionErrorStatus } from "../../types/Encryption";
+import { EncryptionException } from "./CaseEncryptionHelper";
+import { DeviceLocalAESParams } from "./DeviceLocalAESStrategy";
+import {
+  EncryptionContext,
+  EncryptionDependencies,
+  EncryptionPossibility,
+  IEncryptionStrategy,
+} from "./EncryptionStrategy";
+
+const { Aes } = NativeModules;
+
+export interface PasswordParams {
+  password: string;
+}
+
+async function generateAESParamsFromPassword(
+  password: string
+): Promise<DeviceLocalAESParams> {
+  const key = await Aes.pbkdf2(password, "salt4D42bf960Sm1", 5000, 256);
```

Comment:
As mentioned above, put in constants. These values are also in multiple places which makes it even better to have those in constants.